.github/actions/build-wheels/action.yml

```diff
@@ -0,0 +1,73 @@
+name: 'Build Wheels'
+description: 'Build wheels for ofrak'
+inputs:
+  python-version:
+    description: 'Python version to build for'
+    required: true
+  os:
+    description: 'Operating system'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        lfs: false
+
+    - name: Setup LFS cache
+      uses: ./.github/actions/lfs-cache
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ inputs.python-version }}
+        cache: pip
+        cache-dependency-path: |
+          requirements-*.txt
+          **/setup.py
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '18'
+
+    - name: Build frontend
+      shell: bash -euxo pipefail {0}
+      run: make -C ofrak_core src/ofrak/gui/public
+
+    - name: Install build tools
+      shell: bash -euxo pipefail {0}
+      run: |
+        python -m pip install -r requirements-pip.txt
+        python -m pip install -r requirements-build.txt
+
+    - name: Set Python version for cibuildwheel
+      id: py-version
+      shell: bash -euxo pipefail {0}
+      run: |
+        PY_VERSION="${{ inputs.python-version }}"
+        PY_VERSION_NO_DOT=${PY_VERSION//./}
+        echo "version=${PY_VERSION_NO_DOT}" >> $GITHUB_OUTPUT
+
+    - name: Build ofrak wheels with cibuildwheel
+      shell: bash -euxo pipefail {0}
+      env:
+        CIBW_BUILD: "cp${{ steps.py-version.outputs.version }}-*"
+        CIBW_SKIP: "*-win32 *-manylinux_i686"
+        CIBW_ARCHS_WINDOWS: "AMD64"
+        CIBW_ARCHS_MACOS: "x86_64 arm64"
+        CIBW_ARCHS_LINUX: "x86_64"
+        CIBW_BUILD_VERBOSITY: 1
+      run: |
+        cd ofrak_core
+        cibuildwheel --output-dir ../wheelhouse
```

Comment:
Does this automatically find all packages to build wheels for? Is that why we need a bunch of different workflows and jobs to build source distributions, but only one workflow/job (spread across the matrix) to build wheels?